<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GereStock</class>
 <widget class="QDialog" name="GereStock">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gérer les Sorties de Stock </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/img7.jpg);
background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date Sortie</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>id Produit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtQuntite">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtProduit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDateSortie">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>id Client</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Id Produit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtClient">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stock Disponible</string>
   </property>
  </widget>
  <widget class="QLabel" name="ValueStock">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="idValue">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Verfier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quntitéi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
